File: app/src/main/java/com/keltapps/makrokosmos/audio/service/PlaybackInfoListener.java
package com.keltapps.makrokosmos.audio.service;

import android.support.v4.media.session.PlaybackStateCompat;

public abstract class PlaybackInfoListener {

    public abstract void onPlaybackStateChange(PlaybackStateCompat state);

    public void onPlaybackCompleted() {
    }
}

File: app/src/main/java/com/keltapps/makrokosmos/audio/service/PlaybackInfoListener.kt
package com.keltapps.makrokosmos.audio.service

import android.support.v4.media.MediaMetadataCompat
import android.support.v4.media.session.PlaybackStateCompat

abstract class PlaybackInfoListener {

    abstract fun onPlaybackStateChange(state: PlaybackStateCompat, currentMedia: MediaMetadataCompat)

    fun onPlaybackCompleted() {}
}